flow/cmd/mirror_status.go

```diff
@@ -106,6 +105,71 @@ func (h *FlowRequestHandler) CDCFlowStatus(
 	}, nil
 }
 
+func (h *FlowRequestHandler) cloneTableSummary(
+	ctx context.Context,
+	flowJobName string,
+) (*protos.CloneTableSummary, error) {
+	cfg := h.getQRepConfigFromCatalog(flowJobName)
+	res := &protos.CloneTableSummary{
+		FlowJobName: flowJobName,
+		TableName:   cfg.DestinationTableIdentifier,
+	}
+
+	q := `
+	SELECT 
+		MIN(start_time) AS StartTime,
+		COUNT(*) AS NumPartitionsTotal,
+		COUNT(CASE WHEN end_time IS NOT NULL THEN 1 END) AS NumPartitionsCompleted,
+		SUM(rows_in_partition) FILTER (WHERE end_time IS NOT NULL) AS NumRowsSynced,
+		AVG(EXTRACT(EPOCH FROM (end_time - start_time)) * 1000) FILTER (WHERE end_time IS NOT NULL) AS AvgTimePerPartitionMs
+	FROM 
+		peerdb_stats.qrep_partitions
+	WHERE 
+		flow_name = $1
```

Comment:
For snapshot clones, this will be of the form  `clone_<flow_name>_<id>` - resulting in now initial load data being returned.